myapp/test/controllers/tasks_controller_test.rb

```diff
@@ -0,0 +1,39 @@
+require 'test_helper'
+
+class TasksControllerTest < ActionDispatch::IntegrationTest
```

Comment:
It would be better to write a test for each method

for example
index → check that multi of data being listed
edit → check that changed some data
create → check data was registered to db

Reply:
Sure, I have write some test case in step 8.